src/proto/ssl_gc_state.proto

```diff
@@ -0,0 +1,128 @@
+syntax = "proto2";
+
+package SSLProto;
+
+import "proto/ssl_gc_common.proto";
+import "proto/ssl_gc_geometry.proto";
+import "proto/ssl_gc_game_event.proto";
+import "proto/ssl_gc_referee_message.proto";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+message YellowCard
+{
+    optional uint32 id                               = 1;
+    optional GameEvent caused_by_game_event          = 2;
+    optional google.protobuf.Duration time_remaining = 3;
+}
+
+message RedCard
+{
+    optional uint32 id                      = 1;
+    optional GameEvent caused_by_game_event = 2;
+}
+
+message Foul
+{
+    optional uint32 id                           = 1;
+    optional GameEvent caused_by_game_event      = 2;
+    optional google.protobuf.Timestamp timestamp = 3;
+}
+
+message Command
+{
+    required Type type     = 1;
+    optional Team for_team = 2;
+
+    enum Type
+    {
+        UNKNOWN        = 0;
+        HALT           = 1;
+        STOP           = 2;
+        NORMAL_START   = 3;
+        FORCE_START    = 4;
+        DIRECT         = 5;
+        INDIRECT       = 6;
+        KICKOFF        = 7;
+        PENALTY        = 8;
+        TIMEOUT        = 9;
+        BALL_PLACEMENT = 10;
+    }
+}
+
+message GameState
+{
+    required Type type     = 1;
+    optional Team for_team = 2;
+
+    enum Type
+    {
+        UNKNOWN        = 0;
+        HALT           = 1;
+        STOP           = 2;
+        RUNNING        = 3;
+        FREE_KICK      = 4;
+        KICKOFF        = 5;
+        PENALTY        = 6;
+        TIMEOUT        = 7;
+        BALL_PLACEMENT = 8;
+    }
+}
+
+message Proposal
+{
+    // The timestamp when the game event proposal occurred
+    optional google.protobuf.Timestamp timestamp = 1;
+    // The proposed game event.
+    optional GameEvent game_event = 2;
+}
+
+message ProposalGroup
+{
+    // List of proposals in this group
+    repeated Proposal proposals = 1;
+    // Whether the proposal group was accepted
+    optional bool accepted = 2;
+}
+
+message TeamInfo
```

Comment:
this was copy pastad from SSLs repo, one of the team info is local to Referee (Refree::TeamInfo)

and the other one is global but they have some common items

Reply:
anyway to delete one of them?

Reply:
they are not exactly the same protos (some overlap, but they have a lot of different fields)

 I don't think it's worth doing on our end because neither are defined by us
perhaps we should open a PR to  SSL if we care about it